README.md

```diff
@@ -5,6 +5,8 @@ cf\_units
 
 \(C) British Crown Copyright 2015 - 2016, Met Office
 
+Introduction
+------------
```

Comment:
@ajdawson What about this See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers